src/fluree/db/ledger/mutable.clj

```diff
@@ -58,15 +60,38 @@
             (log/warn (str "Flakes purged from block " block))
             true)))
 
+
+(defn- create-block-map
+  "Creates a block-map from a vector of flakes"
+  [db flakes]
+  (go-try
+    (loop [[^Flake flake & r] flakes
+           t-map     {}
+           block-map {}]
+      (if flake
+        (if-let [block (get t-map (.-t flake))]
```

Comment:
Instead of calling the `-t` method directly, can we use the `flake/t` helper function so we don't need the type hint?

Reply:
replaced all ^Flake references (that I found) with the appropriate `flake/` helper function.